sdk/ai/Azure.AI.Inference/src/Telemetry/SingleRecordedResponse.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+using System.Collections.Generic;
+using System.Text.Json;
+
+namespace Azure.AI.Inference.Telemetry
+{
+    internal class SingleRecordedResponse : AbstractRecordedResponse
+    {
+        private readonly string[] _completions;
+        /// <summary>
+        /// Create the instance of Single recorded response.
+        /// </summary>
+        /// <param name="traceContent">If true the messges and function names will be recorded.</param>
+        /// <param name="response"></param>
+        public SingleRecordedResponse(ChatCompletions response, bool traceContent) {
+            IsEmpty = false;
+            Id = response.Id;
+            Model = response.Model;
+            PromptTokens = response.Usage.PromptTokens;
+            CompletionTokens = response.Usage.CompletionTokens;
+
+            _completions = new string[response.Choices.Count];
+            // Record the event for each response
+            int i = 0;
+            // For ChatCompletions we do not have single finish reason
+            // we will take it from the last chat choice.
+            if (response.Choices.Count > 0)
+                FinishReason = response.Choices[response.Choices.Count - 1].FinishReason.ToString();
```

Comment:
finish reason attribute is an array - https://github.com/open-telemetry/semantic-conventions/blob/main/docs/gen-ai/gen-ai-spans.md

<img width="1199" alt="image" src="https://github.com/user-attachments/assets/76fa8a5c-35fe-4f87-8de0-6b73b90d326a">

please populate it as such

Reply:
Fixed.